composer.json

```diff
@@ -18,14 +18,14 @@
   "homepage": "https://github.com/atk4/data",
   "license": "MIT",
   "authors": [
-    {
-      "name": "Romans Malinovskis",
-      "email": "romans@agiletoolkit.org",
-      "homepage": "https://nearly.guru/"
-    }
+    {"name": "Romans Malinovskis", "email": "romans@agiletoolkit.org", "homepage": "https://nearly.guru/" },
+    {"name": "Michael Voříšek", "homepage": "https://mvorisek.cz/"}
```

Comment:
@mvorisek can we set it up so the entire contributor team would get credit?

Reply:
Yes, based on https://github.com/atk4/data/graphs/contributors at least @DarkSide666 for this repo - @DarkSide666, please provide name, home page (and optionally an email) if you want to be added or let me know you do not want to.

Reply:
Yes, I'm absolutely fine with that :)
It would be great if composer.json could read these names from atk4/image or atk4/infrastructure, to always be up to date, but I guess it's not possible.